src/USER-CONP/fix_charge_update.cpp

```diff
@@ -253,6 +265,12 @@ void FixChargeUpdate::setup(int) {
     }
   }
 }
+/* ---------------------------------------------------------------------- */
+
+void FixChargeUpdate::setup(int) {
+  // correct forces for initial timestep
+  short_range_correction(true);
```

Comment:
Ich würde das gaussian_charge_forces() nennen?

Reply:
gausscorr() wäre auch OK.

Reply:
Habe es in gausscorr() umbenannt